src/pages/Login.tsx

```diff
@@ -0,0 +1,199 @@
+// import React from 'react';
```

Comment:
delete commented lines and make sure you format your file with prettier. I can see white spaces at the beggining of the component

Reply:
Fixed